examples/models/export_hf_model.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import os
+import sys
+import traceback
+
+from examples.portable.utils import export_to_edge
+from executorch.backends.xnnpack.partition.xnnpack_partitioner import (
+    XnnpackDynamicallyQuantizedPartitioner,
```

Comment:
this partitioner should only be used with dynamically quantized models, if we're not using the quantization flow, then we should try with XnnpackPartitioner instead.

Reply:
@mcr229 just add a patch to the PR?

Reply:
Switched to use the `XnnpackPartitioner` and rerun the lowering. Only 2 models previously succeeded fail with error
```
error:
  /tmp/tmpty09bomb/data.json:1: 128624: error: cannot parse value starting with: -
``` 
which happens during serialization, by then `XnnpackPartitioner` should have been run through successfully, correct? @mcr229